src/components/util/util.js

```diff
@@ -43,10 +43,11 @@ export const formatDateAndTime = (date, formatter) => {
   return date ? formatter(date, { day: 'numeric', month: 'numeric', year: 'numeric' }) : '';
 };
 
+
 export const getServicePointOfCurrentAction = (action, servicePoints = []) => {
   const servicePoint = servicePoints.find(sp => sp.id === action.createdAt);
-
-  return servicePoint ? servicePoint.name : action.createdAt;
+  const createAtValue = servicePoint ? servicePoint.name : action.createdAt;
+  return createAtValue || '-';
```

Comment:
Please use [`<NoValue>`](https://github.com/folio-org/stripes-components/tree/master/lib/NoValue) instead of `-` for accessibility reasons.

Reply:
@zburke the change has been done.